### ISSUE TITLE
Error message on login does not get read by screen reader.

### ISSUE THREAD

**Issue by piet0024** [gctools-outilsgc/gcconnex#913]
To replicate
- enter username and password that will fail to generate error message
- focus then starts at the top of page, but then instantly follows the autofocus on the username input. This bypasses the error message to the user.

The error message must be read first.

@CoolColin

**Comment by piet0024** [comment]
This is an issue with all the system messages that occur on pageload.

Quick idea to fix (might not work) have the system message be delayed and loaded through AJAX to the messages area. This div has an aria-live that reads messages as they get added to them dynamically.
